README.md

```diff
@@ -4,6 +4,8 @@
 
 ## A fan-made, web-based version of Stonemaier Games' living roll-and-write game
 
+#### Note: This digital implementation is based on [version 10.1 of the original print-and-play Rolling Realms](https://www.dropbox.com/sh/kj5tunjjx1xfhn5/AACc63pMv-erHHfN69_xM2nAa?dl=0&preview=Rolling+Realms+v10.1.pdf). Stonemaier has requested that this implementation remain based on version 10.1, so we will not be adding any new realms outside of the original nine. You can support Stonemaier and play all the new realms digitally with the [Tabletopia implementation of Rolling Realms](https://tabletopia.com/games/rolling-realms), or buy the [physical version](https://stonemaiergames.com/games/rolling-realms/).
```

Comment:
I might say "Stonemaier Games" instead of Stonemaier.

I'd definitely bold that text. I'd also make one change, from:

> we will not be adding any new realms outside of the original nine

to:

> this project will never include any new realms or adjustments to existing realms to achieve parity with the physical version.

If you want to be hypervigilant about this, you can always shoot an email to Jamey and ask for his thoughts on the wording.

Reply:
Just noticing that the whole thing is an `h4`, so it should be bolded

sorry about that! 😬